packer/build-vm.ps1

```diff
@@ -0,0 +1,122 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+<#
+.SYNOPSIS
+    This builds a Starter Kit virtual machine using Packer
+.DESCRIPTION
+    Configures Packer logging, Defines a network adapter and vm switch,
+    compresses assessment PowerShell scripts, and intitiates the packer build.
+.EXAMPLE
+    PS C:\> .\build-vm.ps1
+    Creates a virtual machine image that can be imported using the Hyper-V Manager
+.NOTES
+    Sets the Packer debug mode and logging path variables at runtime.
+#>
+
+param([string] $VMSwitch = "packer-hyperv-iso",
+    [string] $ISOUrl = $null,
+    [switch] $SkipCreateVMSwitch = $false,
+    [switch] $SkipRunPacker = $false)
+
+#Requires -RunAsAdministrator
+#Requires -Version 5
+
+#global vars
+$buildPath = Join-Path -Path $PSScriptRoot -ChildPath "build"
+$logsPath = Join-Path -Path $buildPath -ChildPath "logs"
+$distPath = Join-Path -Path $PSScriptRoot -ChildPath "dist"
+
+function Invoke-CreateFolders {
+    New-Item -ItemType Directory -Path $buildPath -Force | Out-Null
+    New-Item -ItemType Directory -Path $logsPath -force | Out-Null
+    New-Item -ItemType Directory -Path $distPath -force | Out-Null
+}
+
```

Comment:
What do you think about putting a check in here somewhere to make sure you have enough disk space? It would save time to know that the 18GB+ output files are not going to fit after running it for 45 minutes.

Reply:
i set the minimum to 30gb, good idea btw